civihr_default_theme/scss/base/_custom.scss

```diff
@@ -1,3 +1,5 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id, scss/at-extend-no-missing-placeholder */
```

Comment:
i don't know why you are using this, you should use it only for the civicrm/ part of shoreditch